UPGRADE-4.3.md

```diff
@@ -110,3 +110,30 @@ Yaml
 ----
 
  * Using a mapping inside a multi-line string is deprecated and will throw a `ParseException` in 5.0.
+
+Workflow
+--------
+
+ * `MarkingStoreInterface::setMarking()` will have a third argument in Symfony 5.0.
+
+   Before:
+   ```php
+   class MyMarkingStore implements MarkingStoreInterface
+   {
+     public function setMarking($subject, Marking $marking)
+     {
+
```

Comment:
we can remove the empty lines in the method bodies (or replace it with the `// ...` placeholder comment)